tactics/tacticals.mli

```diff
@@ -44,10 +44,8 @@ val tclAT_LEAST_ONCE : tactic -> tactic
 val tclFAIL          : int -> std_ppcmds -> tactic
 val tclFAIL_lazy     : int -> std_ppcmds Lazy.t -> tactic
 val tclDO            : int -> tactic -> tactic
-val tclWEAK_PROGRESS : tactic -> tactic
```

Comment:
They were not officially deprecated, but were used in one very specific case in the Equality module. The other uses are semantically equivalent to `tclPROGRESS`.